herbert_core/classes/MPIFramework/@parallel_config/private/check_and_set_worker_.m

```diff
@@ -9,6 +9,14 @@
 end
 scr_path = which(new_wrkr);
 if isempty(scr_path)
+    % Check if it is a compiled worker
+    new_wrkr = check_compiled_(new_wrkr);
+    if ~isempty(new_wrkr)
+        config_store.instance().store_config(obj, 'worker', new_wrkr);
```

Comment:
`config_store.instance()` is used a lot here, does this want to be instanced and assigned to a variable to avoid repetition?

Reply:
it still works...